adjust.go

```diff
@@ -80,6 +80,32 @@ func AdjustSaturation(img image.Image, percentage float64) *image.NRGBA {
 	})
 }
 
+// AdjustHues changes the hue of the image using the shift parameter (measured in degrees) and returns the adjusted image.
+// The shift must be in the range (-180, 180).
+// The shift = 0 gives the original image.
+// The shift = 180 (or -180) corresponds to a 180° degree rotation of the color wheel and thus gives the image with its hue inverted for each pixel.
+//
+// Examples:
+//  dstImage = imaging.AdjustHues(srcImage, 90) // Shift Hue by 90°.
+//  dstImage = imaging.AdjustHues(srcImage, -30) // Shift Hue by -30°.
+//
+func AdjustHues(img image.Image, shift float64) *image.NRGBA {
+	if shift == 0 {
+		return Clone(img)
+	}
+
+	shift = math.Min(math.Max(shift, -180), 180)
```

Comment:
I think it would be more user-friendly to remove the -180..180 restriction on the shift parameter. E.g.  shifting by 270° is the same as shifting by -90°. What do you think?
Removing this line is probably the only change needed to support this (needs verifying with additional test cases).

Reply:
I tried removing the line, but -350 does not return the same result as +10 (test fails). So this will need further investigation...

Reply:
Fixed by commit 3956733